extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -0,0 +1,12 @@
+amp-story-audio-sticker {
+  z-index: 3 !important;
+  pointer-events: visible !important;
+
+  position: absolute;
+  max-width: 100%;
```

Comment:
Didn't you want to set a max size that's much smaller? Also, you'd probably want `overflow: hidden` if you set a max size, otherwise it doesn't affect the size of the children. And, use the CSS vars for the page size, otherwise someone can put this in a div with `width: 300%` and basically make it as wide as they want (though it wouldn't matter too much if we allow these to have size of 100%)

Reply:
Correct. Will do that in future PRs. Thanks for the reminder! Remove this for now.

Btw why could a CSS var help avoid the issue of the huge div wrapper(e.g. width: 300%) as you mentioned?